client/src/components/manageProjects/editProject.js

```diff
@@ -79,174 +137,73 @@ const EditProject = ({
-                   </div>
-                <div className="event-list-description">{`${event.description}`}</div>
-              </button>
-            </li>
-          ))}
-        </ul>
-      </div>
-      <div className="display-events">
-        <button
-          type="button"
-          className="create-form-button"
-          onClick={() => setIsCreateNew(true)}
+      <Box sx={{ bgcolor: '#F5F5F5', my: 2 }}>
+        <Box
+          sx={{
+            p: 2,
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+          }}
         >
-          Create New Event
-        </button>
-      </div>
-
-      <div className="event-list">
-        <h3>Manually Edit Events Checkin</h3>
-        <h2 className="event-alert">{eventAlert}</h2>
-        <ul>
-          {regularEventsState.map((event, index) => (
-            
-            // eslint-disable-next-line no-underscore-dangle
-            <RegularEvent event={event} key={event._id} updateRegularEvent={updateRegularEvent} />
-          ))}
-        </ul>
-      </div>
-    </div>
+          <Box>
+            <Typography sx={{ fontSize: '18px', fontWeight: '600' }}>
+              Recurring Events
+            </Typography>
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              '&:hover': { color: 'red', cursor: 'pointer' },
+            }}
+            onClick={() => setIsCreateNew(true)}
+          >
+            <PlusIcon style={{ marginRight: '7px' }} />
+            <Typography
+              sx={{
+                fontSize: '14px',
+                fontWeight: '600',
+              }}
+            >
+              Add New Event
+            </Typography>
+          </Box>
+        </Box>
+        <Divider sx={{ borderColor: 'rgba(0,0,0,1)' }} />
+        <Box sx={{ py: 2, px: 4 }}>
+          <div className="event-list">
+            <h2 className="event-alert">{eventAlert}</h2>
+            <ul>
+              {rEvents.map((event) => (
+                // eslint-disable-next-line no-underscore-dangle
+                <li key={`${event.event_id}`}>
+                  <button type="button" onClick={() => setSelectedEvent(event)}>
+                    <div>{event.name}</div>
+                    <div className="event-list-details">
+                      {`${event.dayOfTheWeek}, ${event.startTime} - ${event.endTime}; ${event.eventType}`}
+                      <div className="edit-icon">
+                        <EditIcon cursor="pointer" />
+                      </div>
+                    </div>
+                    <div className="event-list-description">{`${event.description}`}</div>
+                  </button>
+                </li>
+              ))}
+            </ul>
+          </div>
+          <div className="display-events"></div>
+        </Box>
+      </Box>
+      <Box sx={{ textAlign: 'center' }}></Box>
+    </Box>
   );
 };
 
```

Comment:
Can you restore this section, removing it is beyond the scope of this pull request. Don't worry about styling, just restore the function and render below the rest of the elements.

Reply:
Can you please clarify on this. Did you want me to undo changes to the Recurring Events section?  If so, I just want to clarify that the changes were only to the UI and there were no functionality changes. I applied the UI since the issue was to apply the Figma design to the Edit page which also had the recurring events included but I can revert back to its original.

Reply:
There is a temporary fix in here so that if an Admin needs to they can manually change an event to CheckInReady. It's a temporary fix while we figure out what the issue with the recurring event checkin bugs are.